tests/spread/integration/openjdk-8-jre-headless/smoke.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+set -ex
+
+for java in `find /usr/lib/jvm -name java`; do
+# workaround missing /proc filesystem
```

Comment:
can you explain this? 

The test already runs inside a container, which is our typical and existing use case nonetheless. so why do we need this additional step here, given that `/proc` is a special directory inside the container?

Reply:
We create a chroot with openjdk slice and this chroot does not have /proc. 
This breaks loading dynamic libraries with a relative path (e.g. $ORIGIN/...) as the loader checks /proc/self/exe to discover the value of $ORIGIN. 
The chroot should either have a /proc filesystem - for this we need to install mount slice and mount /proc after (install-slices) inside the chroot or do the workaround like this.

Reply:
Also checked whether I can mount/bind-mount /proc - it seems that container lacks permissions for mounting /proc.